lib/rubocop/cop/rails/reversible_migration.rb

```diff
@@ -129,6 +129,51 @@ module Rails
       #     end
       #   end
       #
+      # @example
+      #   # remove_columns
+      #
+      #   # bad
+      #   def change
+      #     remove_columns :users, :name, :email
+      #   end
+      #
+      #   # good
+      #   def change
+      #     reversible do |dir|
+      #       dir.up do
+      #         remove_columns :users, :name, :email
+      #       end
+      #
+      #       dir.down do
+      #         add_column :users, :name, :string
+      #         add_column :users, :name, :email
+      #       end
+      #     end
+      #   end
+      #
+      #   # good (Rails >= 6.1)
```

Comment:
Rails 6.1 has not been released yet. Can you provide a PR of rails/rails repo that includes this feature?

Reply:
Yeah just found it by chance in [this comment](https://github.com/rubocop-hq/rubocop-rails/issues/110#issuecomment-640359177).

https://github.com/rails/rails/pull/36589

Reply:
@koic Did you just want to see the PR or did you want me to actually add to the comment in code here?

Reply:
@koic updated.